lib/odbc_adapter/database_statements.rb

```diff
@@ -27,7 +27,7 @@ def insert(arel, name = nil, pk = nil, id_value = nil, sequence_name = nil, bind
     # Executes +sql+ statement in the context of this connection using
     # +binds+ as the bind substitutes. +name+ is logged along with
     # the executed +sql+ statement.
-    def internal_exec_query(sql, name = 'SQL', binds = [], prepare: false) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
Function args changed here: https://github.com/rails/rails/commit/eabcff22a86213521eecafbf2dbf3ad82714f941#diff-88e231c48eb5559ac3be58a024b019e86f7c74d388f5d49fac3f4e47bf851cc6R516

Reply:
oooOOOooo, I think I missed this.